packages/@aws-cdk/aws-codecommit/README.md

```diff
@@ -37,6 +37,30 @@ To add an Amazon SNS trigger to your repository:
 repo.notify('arn:aws:sns:*:123456789012:my_topic');
 ```
 
+## Initialize with Code
+
+It is possible to initialize the Repository with an Asset.
+For learning how assets work, see the CDK Assets documentation.
+Be aware, that this will only work on *directories*, not on single files.
+This is due to the fact, that CodeCommit expects a zip in the S3 Bucket.
+
+Example:
+
+```ts
+const readmeAsset = new assets.Asset(this, 'ReadmeAsset', {
+      path: path.join(__dirname, 'directory/'),
+});
+
+const repo = new Repository(this, 'Repository', {
```

Comment:
Code here should use the `codecommit` prefix (see examples above).

Reply:
Maybe we should make it consistent with the contributing guide:
https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#recommendations
`Types from the documented module should be un-qualified`